instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/config.go

```diff
@@ -74,3 +79,17 @@ func WithCommandAttributeDisabled(disabled bool) Option {
 		cfg.CommandAttributeDisabled = disabled
 	})
 }
+
+// CommandSanitizer defines a function that sanitizes a MongoDB command.
+type CommandSanitizer func(command bson.Raw) string
+
+// WithCommandAttributeSanitizer specifies a function to sanitize the MongoDB command.
+func WithCommandAttributeSanitizer(sanitizer CommandSanitizer) Option {
```

Comment:
I am not sure if the name is great as this is only a use case how this option can be used. It could be even used to enrich the data 🤷 Basically defines a function wich is resposible for setting the `db.statment` value. `WithCommandAttributeFunc`? I cannot come up with any good name at the moment.

Reply:
That's true - maybe `WithCommandAttributeTransformer`?